src/realm/sync/noinst/migration_store.cpp

```diff
@@ -101,7 +103,7 @@ bool MigrationStore::is_migrated()
     return m_state == MigrationState::Migrated;
 }
 
-std::string_view MigrationStore::get_query_string()
+const std::string& MigrationStore::get_query_string()
```

Comment:
I pointed this out in a different PR, but I think this actually needs to be a `std::string` rather than a `const std::string&` since m_query_string is protected by a mutex and we don't hold the mutex for the lifetime of the reference returned by this function.